code/part-two/processor/tests/03-CreateCollection.js

```diff
@@ -24,19 +28,17 @@ describe('Create Collection', function() {
     context = new Context();
     txn = new Txn({ action: 'CREATE_COLLECTION' });
     publicKey = txn._publicKey;
-    address = getAddress.collection(publicKey);
+    address = getCollectionAddress(publicKey);
   });
 
-  it('should create a Collection at the correct address', function() {
+  it('should create collections at the correct address', function() {
```

Comment:
I think `address` should be plural too (`should create collections at correct addresses`)